exercise_0/fileMode/FileMode.cpp

```diff
@@ -0,0 +1,22 @@
+//
+// Created by emyrmak on 7/3/2019.
+//
+
+#include "FileMode.h"
+#include <iostream>
+#include <fstream>
+
+int FileMode::run(const string &path) {
+    ifstream  file;
```

Comment:
Future development/a bit more advanced comment ahead:

It's a good programming practice to use abstractions in favour of the actual implementation (that's why dependency injection is so commonly used in C++ because of that and *dependency inversion* rule from SOLID). Such also makes testing easier (if not possible after all).
From my perspective, it would be beneficial to introduce an interface responsible for reading from any resource. Then inject any implementation - in that very case - basing on std::fstream.

However, as mentioned - that's rather out of the scope of that training, so treat that only as an advice for the future. 
In case that would need a bit of more explanation, don't hesitate to reach me with specific questions.